src/components/Footer/index.js

```diff
@@ -37,62 +39,109 @@ export const OfficeStyled = styled(Row)`
+      <Grid>
+        <Row>
+          <Col width={1}>
+            <Padding bottom={3}>
+              <SectionTitle reverse>Find us</SectionTitle>
+            </Padding>
+          </Col>
+        </Row>
+        <Padding bottom={{ smallPhone: 3.5, tablet: 5 }}>
+          <OfficeStyled>
+            <Locations>
+              {data =>
+                data.map(location => {
+                  const streetAddress = location.node.streetAddress.streetAddress.split(
+                    '\n'
+                  )
+
+                  return (
+                    <Office
+                      key={location.node.id}
+                      {...location.node}
+                      streetAddress={streetAddress}
+                    />
+                  )
+                })
+              }
+            </Locations>
+          </OfficeStyled>
+        </Padding>
+      </Grid>
+    </Padding>
+    <FollowUs>
+      <Padding top={3} bottom={3}>
         <Grid>
           <Row>
             <Col width={1}>
-              <Padding bottom={3}>
-                <SectionTitle reverse>Find us</SectionTitle>
-              </Padding>
+              <Social>
+                {social.map(s => (
+                  <li key={s.label}>
+                    <ExternalAnchor href={s.link}>
+                      <img src={s.img} alt={s.label} />
+                    </ExternalAnchor>
+                  </li>
+                ))}
+                <DesktopGDPR>{gdprLinksElement}</DesktopGDPR>
+              </Social>
             </Col>
           </Row>
-          <Padding bottom={{ smallPhone: 3.5, tablet: 5 }}>
-            <OfficeStyled>
-              <Locations>
-                {data =>
-                  data.map(location => {
-                    const streetAddress = location.node.streetAddress.streetAddress.split(
-                      '\n'
-                    )
-
-                    return (
-                      <Office
-                        key={location.node.id}
-                        {...location.node}
-                        streetAddress={streetAddress}
-                      />
-                    )
-                  })
-                }
-              </Locations>
-            </OfficeStyled>
-          </Padding>
         </Grid>
       </Padding>
-      <FollowUs>
-        <Padding top={3} bottom={3}>
-          <Grid>
-            <Row>
-              <Col width={1}>
-                <Social>
-                  {social.map(s => (
-                    <li key={s.label}>
-                      <ExternalAnchor href={s.link}>
-                        <img src={s.img} alt={s.label} />
-                      </ExternalAnchor>
-                    </li>
-                  ))}
-                </Social>
-              </Col>
-            </Row>
-          </Grid>
-        </Padding>
-      </FollowUs>
-    </GreyFooter>
-  )
-}
+    </FollowUs>
+    <Grid>
+      <Row>
+        <Col width={1}>
+          <Padding bottom={3}>
```

Comment:
I think we agreed that we weren't going to use this component any more & that we would slowly remove `styled-components-spacing`'s components from the codebase whenever we touched an old file.

Reply:
Not using anymore in my code as using 1 component but I didn't know about the `styled-components-spacing` not being used anymore. I found that the way we use CSS and the settings we have (see my comment on `Col` and `Row`) would need more documentation.

Reply:
I believe that we agreed this on a call between you me and David, that we would slowly remove `Padding` components from the codebase. I thought you seemed as excited as me :sweat_smile:

Reply:
Haha! I don't like these `Row` `Col` `Padding` components as I don't find them explicit enough but I don't remember what we agreed. I feel like Dory from Finding Nemo right now; short-term memory 😆

Reply:
Yeah I don't blame you :sweat_smile: